Mathlib/Combinatorics/SetFamily/Compression/UV.lean

```diff
@@ -334,7 +334,7 @@ theorem shadow_compression_subset_compression_shadow (u v : Finset α)
           sup_sdiff_mem_of_mem_compression Ht hvt <| disjoint_of_erase_right hau huvs.1, a, ?_, ?_⟩
       · rw [sup_eq_union, mem_sdiff, mem_union]
         exact ⟨Or.inl hat, hav⟩
-      · rw [← erase_sdiff_comm, sup_eq_union, erase_union_distrib, erase_eq_of_notMem hau]
+      · simp [← erase_sdiff_comm, erase_union_distrib, erase_eq_of_notMem hau]
```

Comment:
(Note to future readers: this is because `sup_eq_union` now has arguments, so it only gets rewritten once.)